src/Relation/Unary.agda

```diff
@@ -266,6 +266,11 @@ P ⊥′ Q = P ∩ Q ⊆′ ∅
 _⊢_ : (A → B) → Pred B ℓ → Pred A ℓ
 f ⊢ P = λ x → P (f x)
 
+-- Pushforward.
```

Comment:
I think a certain amount of the discussion on the PR should go in comments here. At least that would give a hope of someone finding this via just plain 'grep'.

Reply:
@JacquesCarette I re-requested a review from you (towards merging this as-is), so that we can move the discussion on 'more commentary' to #2866